tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -300,6 +300,13 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     csinfo_.mkl_fused_conv2d = "_MklFusedConv2D";
     csinfo_.mkl_fused_depthwise_conv2d = "_MklFusedDepthwiseConv2dNative";
     csinfo_.mkl_fused_matmul = "_MklFusedMatMul";
+    csinfo_.mkl_native_conv2d_with_bias = "_MklNativeConv2DWithBias";
```

Comment:
I remember we have tired that before. The problem is _MklNative* ops are used in both graph and eager modes. Eager runtime always checks for # of inputs and outputs to match the op def. Since we have twice as the number of inputs/outputs with the current ops, the check would fail. What do you think?

Reply:
Ah. I forgot about that. This is fine then. We can try eliminating the MKL second tensors in the new TFRT+MLIR stack. Thank you for the clarification!